build/webpack.common.config.js

```diff
@@ -65,7 +65,8 @@ module.exports = {
 					plugins: [
 						// converts `export default 'foo'` to `exports.default = 'foo'`
 						require.resolve('babel-plugin-add-module-exports'),
-
+						// rewrites Promise references to es6-promise, but only if necessary
```

Comment:
could you add a `//HACK comment` as we really shoudl get to the bottom of why this is happening